src/_pytest/config/__init__.py

```diff
@@ -953,7 +953,7 @@ class ArgsSource(enum.Enum):
         #: Command line arguments.
         ARGS = enum.auto()
         #: Invocation directory.
-        INCOVATION_DIR = enum.auto()
+        INVOCATION_DIR = enum.auto()
```

Comment:
If there are backwards-compatibility concerns, we could also add an alias:

```py
INCOVATION_DIR = INVOCATION_DIR
```

Reply:
Yes, we should add an alias, otherwise it'd be a breaking change.